### PR TITLE
fix: [APPAI-1856] Cleanup deployment templates and correct entry script

### PR DESCRIPTION
**Pull request by dgpatelgit**
# Description

PR includes:
- Need to generate image in new quay.io/app-sre repository. 
- Cleanup of deployment templates and 
- Fixed entry script pythonpath issues.

Fixes # (issue)
Jira ticket ;: https://issues.redhat.com/browse/APPAI-1856

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
 
-zip -r /tmp/training.zip /src
 mkdir -p /tmp/hpf/
 
-gunicorn --pythonpath /src -b 0.0.0.0:6006 --workers=1 -k sync -t 900 flask_endpoint:app
+gunicorn --pythonpath /opt/app-root/src/src -b 0.0.0.0:6006 --workers=1 -k sync -t 900 flask_endpoint:app --log-level $FLASK_LOGGING_LEVEL
```
Comment (arajkumar) [inline entrypoint.sh:5]: Not necessary to give the absolute path, please check https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/scripts/entrypoint.sh

File: openshift/template.yaml [hunk 2]
```diff
@@ -8,91 +8,94 @@ metadata:
     description: f8a-hpf-insights
```
Comment (arajkumar) [inline openshift/template.yaml:8]: Actually you should have referred the following template from app-interface
https://github.com/fabric8-analytics/f8a-hpf-insights/blob/master/openshift/template-prod.yaml

**Review by arajkumar — CHANGES_REQUESTED** [review]

File: openshift/template.yaml (hunk 2 above)
Comment (dgpatelgit) [reply openshift/template.yaml:8]: @arajkumar It is the same template I renamed, template-prod.yaml to template. I don't see any purpose of having old template.yaml file?

Comment (arajkumar) [reply openshift/template.yaml:8]: lgtm

**Comment by arajkumar** [comment]
lgtm

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**